cmd/minikube/cmd/start.go

```diff
@@ -1304,10 +1304,20 @@ func validateFlags(cmd *cobra.Command, drvName string) { //nolint:gocyclo
 		validateCNI(cmd, viper.GetString(containerRuntime))
 	}
 
+	if viper.GetBool(preloadWindowsIso) {
+		if err := download.WindowsISO(constants.DefaultWindowsServerIsoURL); err != nil {
+			exit.Message(reason.Usage, "Failed to download Windows ISO file: {{.constants.DefaultWindowsServerIsoURL}}", out.V{"constants.DefaultWindowsServerIsoURL": constants.DefaultWindowsServerIsoURL})
+		}
+	}
+
 	if cmd.Flags().Changed(windowsNodeVersion) {
 		if err := validateWindowsOSVersion(viper.GetString(windowsNodeVersion)); err != nil {
 			exit.Message(reason.Usage, "{{.err}}", out.V{"err": err})
 		}
+
+		if err := download.WindowsISO(constants.DefaultWindowsServerIsoURL); err != nil {
```

Comment:
This looks like it will always download the same ISO regardless of what windows node version I set

Reply:
made a change to address this!